CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ set_target_properties(tinyxml2 PROPERTIES
 if(DEFINED CMAKE_VERSION AND NOT "${CMAKE_VERSION}" VERSION_LESS "2.8.11")
     target_include_directories(tinyxml2 PUBLIC 
                           $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
-                          $<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>)
+                          $<INSTALL_INTERFACE:include>)

```

Comment:
Since GNUInstallDirs used you can do `CMAKE_INSTALL_INCLUDEDIR` here.

Reply:
Done, thanks. This is relative.